src/core/agent/services/signifyNotificationService.ts

```diff
@@ -333,7 +329,7 @@ class SignifyNotificationService extends AgentService {
     });
   }
 
-  private markNotification(notiSaid: string) {
+  markNotification(notiSaid: string) {
```

Comment:
This can be removed, and we can just call `deleteNotificationRecordById` which does both the marking and deleting in our local DB. Otherwise we might forgot to mark the notification somewhere in the code.